src/core/url_store/atoms/urlStore.ts

```diff
@@ -61,15 +62,21 @@ export const urlStoreAtom = createAtom(
 
         // Apply map position
         if (initialState.bbox) {
-          actions.push(currentMapPositionAtom.setCurrentMapBbox(initialState.bbox));
+          actions.push(
+            v3ActionToV2(setCurrentMapBbox, initialState.bbox, 'setCurrentMapBbox'),
+          );
         } else if (initialState.map) {
           actions.push(
-            currentMapPositionAtom.setCurrentMapPosition({
-              // adjustments performed in url decoder
-              zoom: Number(initialState.map[0]),
-              lat: Number(initialState.map[1]),
-              lng: Number(initialState.map[2]),
-            }),
+            v3ActionToV2(
+              setCurrentMapPosition,
+              {
+                // adjustments performed in url decoder
+                zoom: Number(initialState.map[0]),
```

Comment:
Is it better to unpack these values like `const [zoom, lat, lng] = initialState.map`?